source/extensions/quic_listeners/quiche/platform/quic_hostname_utils_impl.h

```diff
@@ -23,7 +23,7 @@ class QUIC_EXPORT_PRIVATE QuicHostnameUtilsImpl {
 
   // Convert hostname to lowercase and remove the trailing '.'.
   // WARNING: mutates |hostname| in place and returns |hostname|.
-  // NOTE(wub): This only removes trailing dots for now.
+  // NOTE(wub): This only does lowercasing and removes trailing dots for now.
```

Comment:
// Convert hostname to lowercase and remove the trailing '.'.
// NOTE(wub): This only does lowercasing and removes trailing dots for now.

can we remove the note?

Reply:
Rewrote the comment to reflect the current state. PTAL.